### DISCUSSION TITLE
Example how to call rust functions using multiple parameters

### DISCUSSION THREAD

**Discussion by tpisto** [penberg/rusty_jsc#4] [Q&A]
Didn't found good examples about that.

**Comment by penberg** [comment]
I didn't implement that yet. If you look at the `callback` macro:

https://github.com/penberg/rusty_jsc/blob/main/macros/src/lib.rs#L19

you will see that arguments are passed in `_arguments` in the callback trampoline, but never passed to the callback Rust function.

Easy fix is to just pass the arguments as an array. A better solution is to map the arguments array to Rust function arguments, but that's bit beyond my Rust macro skills right now.
